_plugins/googlescholar.rb

```diff
@@ -6,8 +6,8 @@
 
 module Jekyll
   class ScholarStats < Generator
-    # ID for Bungartz et al. paper
-    CITATION_ID = '5053469347483527186'.freeze
+    # ID for PreCICE v2 paper
```

Comment:
Also important for consistency: `preCICE` instead of `PreCICE`. We always use `preCICE` where capitalization is expected (even in the beginning of a sentence), `precice` where not (repositories, code, etc).

Read more: https://precice.org/dev-docs-naming.html#the-precice-name